mkinitcpio

```diff
@@ -405,6 +406,11 @@ while :; do
             shift
             _optmoduleroot=$1
             ;;
+        -D|--hookdir)
+            shift
+            _d_hooks="$(readlink -f "$1")/hooks"
+            _d_install="$(readlink -f "$1")/install"
```

Comment:
Why use readlink on these? Also, why not allow for multiple dirs?

Reply:
Always felt proper to resolve the complete paths when taking user input.

And my use case doesn't involve multiple dirs, I think, so went the easy route.

Reply:
If you aren't going to do error checking on readlink, I don't particularly see the point. Dropping it also makes it simpler to support multiple paths.

Reply:
It is user input anyway. I agree with @falconindy, unless we error check this, I don't see the need for readlink.

Reply:
I had an attempt doing multiple hook paths, dropping `readlink` in the process.